src/adapter/adapter.ts

```diff
@@ -1,93 +1,20 @@
-  .addMapper(priceFilterMapper, 'numberRangeFilter')
-  .replaceClassRequestMapper(SearchRequestMapper)
-  .setFeatureConfig('search', {
-    endpoint: '{env}/query/empathy/search',
-    responsePaths: {
-      results: 'catalog.content',
-      facets: 'catalog.facets',
-      totalResults: 'catalog.numFound',
-      spellcheck: 'catalog.spellchecked',
-      banners: 'banner.content',
-      promoteds: 'promoted.content',
-      redirections: 'direct.content',
-      queryTagging: 'catalog.tagging.query'
-    }
-  })
-  .setFeatureConfig('searchById', {
-    endpoint: '{env}/query/empathy/skusearch',
-    responsePaths: {
-      results: 'catalog.content'
-    }
-  })
-  .setFeatureConfig('relatedTags', {
-    endpoint: '{env}.empathy.co/relatedtags/empathy',
-    responsePaths: {
-      relatedTags: 'data.relatedtags'
-    }
-  })
-  .setFeatureConfig('nextQueries', {
-    endpoint: '{env}.empathy.co/nextqueries/empathy',
-    responsePaths: {
-      nextQueries: 'data.nextqueries'
-    }
-  })
-  .setFeatureConfig('suggestions', {
-    endpoint: '{env}/query/empathy/empathize',
-    responsePaths: {
-      suggestions: 'topTrends.content'
-    }
-  })
-  .setFeatureConfig('topRecommendations', {
-    endpoint: '{env}/query/empathy/topclicked',
-    responsePaths: {
-      results: 'topclicked.content'
-    }
-  })
-  .setFacetConfig({ modelName: 'HierarchicalFacet' }, 'categoryPaths')
-  .setFacetConfig({ modelName: 'NumberRangeFacet' }, 'price')
-  .configureContainer(container => {
-    container.unbind(DEPENDENCIES.ResponseMappers.facets);
-    container.bind(DEPENDENCIES.ResponseMappers.facets).to(EmpathyFacetMapper);
-    container.bind(DEPENDENCIES.ResponseMappers.facets).to(HierarchicalFacetMapper);
-    container.bind(DEPENDENCIES.ResponseMappers.facets).to(EmpathyNumberRangeFacetMapper);
-    container.bind(DEPENDENCIES.ResponseMappers.facets).to(EmpathySimpleFacetMapper);
-    container.bind(DEPENDENCIES.RequestMappers.Parameters.filtersValue).to(RequestFiltersMapper);
-    container.bind(DEPENDENCIES.requestMappers).to(EmpathyRequestParamsMapper);
-    container.rebind(DEPENDENCIES.endpointsService).to(EmpathyEndpointsService);
-  })
-  .setResultTrackingConfig({
-    click: 'tagging.click',
-    add2cart: 'tagging.add2cart',
-    checkout: 'tagging.checkout'
-  })
-  .onBeforeRequest(({ request }) => {
-    delete request.scope;
-  }, 'topRecommendations')
-  .onBeforeRequest(({ request }) => {
-    if (request.scope === 'none') {
-      delete request.scope;
-    }
-  }, 'search')
-  .setInstance('platform')
-  .build();
+export const adapter = platformAdapter;
+
+/* Code sample about how to extend the result mapper with more fields. */
+
+interface EmpathyDemoPlatformResult extends PlatformResult {
+  season: string;
+}
+
+declare module '@empathyco/x-types' {
+  export interface Result {
+    season: string;
+  }
+}
+
+resultSchema.$override<EmpathyDemoPlatformResult, Partial<Result>>({
+  season: 'season'
+});
```

Comment:
What if we override the `recommendationsRequestMutableSchema` to make the topClicked work for now? Something like:

```
recommendationsRequestMutableSchema.$override<
  RecommendationsRequest,
  PlatformRecommendationsRequest
>({
  start: 'start',
  rows: 'rows',
  origin: 'origin',
  extraParams: ({ extraParams }) => {
    const extraParamsCopy = { ...extraParams };
    delete extraParamsCopy.scope;
    return extraParamsCopy;
  }
});
```

With a TODO to remove this once this issue is solved?

Reply:
Good idea, didn't thought about that copy.